server/controllers/RequestController.js

```diff
@@ -0,0 +1,50 @@
+import { requests } from '../dummydatabase/dummydatabase';
+/**
+  * @class RequestController
+  * @description CRUD operations on Requests
+  */
+class RequestController {
+  /**
+        * @static
+        * @param {object} req - The request payload sent to the router
+        * @param {object} res - The response payload sent back from the controller
+        * @returns {object} - status Message and the particular request by id.
+        * @description This method gets a request by id in maintenance tracker
+        * @memberOf RequestController
+        */
+  static getRequestById(req, res) {
+    const id = req.params.requestId;
```

Comment:
Is there a reason why you are passing the id as param is instead of request body?